docs/SceneWalkthrough/PlayerDataStorageWalkthrough.md

```diff
@@ -0,0 +1,12 @@
+## **Player Data Storage Demo**
+This demo showcases the Player Data Storage interface. This allows games to store player specific data in cloud servers. The demo uses a faux inventory to showcase this.
+- The ``Enter filename`` box is used to enter a filename to be created locally.
+- The ``New Inventory File`` creates an inventory file with the specified name in the cloud.
+- The ``Refresh``, ``Duplicate``, and ``Delete`` buttons all do their respective action to the selected file in the window below.
+- The ``Files:`` window displays inventory files in the cloud, selecting one will display its details in the ``Remote View`` window.
+- The ``Download`` button downloads the currently selected file to the disk.
+- The ``Add Item`` add the item from the dropdown to the current local inventory.
+- The ``Upload`` button uploads the current local inventory file to the cloud, overwriting the old version.
+
```

Comment:
do you need to do things in the developer portal to make this work? if so might need a note on that

Reply:
from my understanding the client generates the file, the only options in the dev portal are to look at individual player's "uploads"